tools/generator/src/Generator.swift

```diff
@@ -131,7 +131,10 @@ Was unable to merge "\(srcTarget.label) \
             pbxTargets
         )
 
-        let schemes = environment.createXCSchemes(disambiguatedTargets)
+        let schemes = try environment.createXCSchemes(
+            filePathResolver,
+            pbxTargets
```

Comment:
@brentleyjones I just realized that the `Bazel Dependencies` target is not included in the `pbxTarget`. Should we add it to the `Dictionary` or should I pass along the `Bazel Dependencies` target as a special case?

Reply:
It can be added to `pbxTargets` before `addTargets()` returns it. Since we only use it as a looked elsewhere, it won't have any harm.

Reply:
@brentleyjones Are you OK with me changing the signature for `addTargets` from

```swift
    static func addTargets(
        in pbxProj: PBXProj,
        for disambiguatedTargets: [TargetID: DisambiguatedTarget],
        products: Products,
        files: [FilePath: File],
        filePathResolver _: FilePathResolver,
        bazelDependenciesTarget: PBXAggregateTarget?
    ) throws -> [TargetID: PBXNativeTarget] {
```
 
to 

```swift
    static func addTargets(
        in pbxProj: PBXProj,
        for disambiguatedTargets: [TargetID: DisambiguatedTarget],
        products: Products,
        files: [FilePath: File],
        filePathResolver _: FilePathResolver,
        bazelDependenciesTarget: PBXAggregateTarget?
    ) throws -> [TargetID: PBXTarget] {
```

? A `PBXAggregateTarget` is a `PBXTarget`, but it is not a `PBXNativeTarget`.

Reply:
If so, I propose that we change all of the signatures to take `[TargetID: PBXTarget]`. If you are good with that, I will do that in a separate PR.

Reply:
Yeah, that's dine.